terraform/gcp/variables.tf

```diff
@@ -1,40 +1,97 @@
-variable "user" {
-  description = "Instance user name that will used for SSH operations"
+variable "project" {
+  description = "Name of GCP project that will be used for housing require infrastructure"
```

Comment:
... "the required infrastructure"

Reply:
The description would be "Name of GCP project that will be used for the required infrastructure"